client/landing/gutenboarding/onboarding-block/design-selector/index.tsx

```diff
@@ -112,6 +111,7 @@ const DesignSelector: FunctionComponent< Props > = ( { showPageSelector = false
 				className={ classnames( 'design-selector__grid-container', {
 					'is-page-selector-open': showPageSelector,
 				} ) }
+				tabIndex={-1}
```

Comment:
Should be surrounding space here - did prettier not handle this?

```suggestion
				tabIndex={ -1 }
```

Reply:
It was applied through GitHub's commit suggestion button, no prettier ran there. Better to do it locally next time!